README.md

```diff
@@ -17,35 +17,55 @@ Tramway use [Tailwind](https://tailwindcss.com/) by default. All UI helpers impl
 
 ### Navbar
 
-#### tailwind_link_to
+Tramway provides DSL for rendering Tailwind Navgiation bar.
 
-Tramway provides `tailwind_link_to` method, that uses `link_to`
-
-```haml
-= tailwind_link_to "/users/sign_in" do
-  Sign In
+```ruby
+tramway_navbar brand: 'Purple Magic' do |nav|
```

Comment:
is `brand` the right word in this context? Maybe `title` would be more common

Reply:
`brand` It's something from the Bootstrap world in 10's )) by the way, you're right. `title` is better

Reply:
Changed